internal/database/inmemory.go

```diff
@@ -1,11 +1,8 @@
 package database
```

Comment:
can you enable the commented out test in `unit_tests.go`?

Reply:
Determined a broken test, not blocking